aws/resource_aws_spot_fleet_request.go

```diff
@@ -256,6 +256,30 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+						"tag": {
```

Comment:
I'm curious if you were able to examine other resource's support of EC2 tags before coming to this implementation? Most resources use our helper file aws/tags.go:

https://github.com/terraform-providers/terraform-provider-aws/blob/494221fd522c1d41c47790e1a04ce78fcd1c3ab6/aws/tags.go#L20-L25

Example usage from `aws_instance`:

https://github.com/terraform-providers/terraform-provider-aws/blob/494221fd522c1d41c47790e1a04ce78fcd1c3ab6/aws/resource_aws_instance.go#L262-L264